### PR TITLE
Spanish translate updated

### PR DESCRIPTION
**Pull request by yayitazale**
## Description
Updated the spanish translation.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] I verified that my code applies to the guidelines (`composer kimai:code-check`)
- [ ] I updated the documentation (see [here](https://github.com/kimai/www.kimai.org/tree/master/_documentation))
- [X] I agree that this code is used in Kimai and will be published under the [MIT license](https://github.com/kevinpapst/kimai2/blob/master/LICENSE)

### CODE REVIEW

**Comment by kevinpapst** [comment]
Thanks @yayitazale four your contribution!
Do you plan to update the other files as well? Just asking to know whether I should wait with the merge?

File: translations/messages.es.xliff [hunk 1]
```diff
@@ -756,6 +946,18 @@
...
+                <source>label.exported</source>
+                <target>Exportado</target>
+            </trans-unit>
+            <trans-unit id="entryState.exported">
+                <source>entryState.exported</source>
+                <target>Tratado</target>
+            <trans-unit id="entryState.not_exported">
+                <source>entryState.not_exported</source>
+                <target>Abierto</target>
+            </trans-unit>
```
Comment (kevinpapst) [inline translations/messages.es.xliff]: this line needs to go up to 956

**Comment by kevinpapst** [comment]
Awesome, then I will keep the PR open. Thanks a lot!

**Comment by yayitazale** [comment]
> 
> 
> Could you have a look at that one minor issue?

Done.

File: translations/actions.es.xliff [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="es" datatype="plaintext" original="none">
```
Comment (kevinpapst) [inline translations/actions.es.xliff]: original="actions.en.xliff"

File: translations/plugins.es.xliff [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="es" datatype="plaintext" original="none">
```
Comment (kevinpapst) [inline translations/plugins.es.xliff]: original="plugins.en.xliff"

File: translations/tags.es.xliff [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="es" datatype="plaintext" original="none">
```
Comment (kevinpapst) [inline translations/tags.es.xliff]: original="tags.en.xliff"

**Comment by kevinpapst** [comment]
I will include the theme update in another PR which I will merge later today

**Comment by kevinpapst** [comment]
Thanks 🎉

**Comment by yayitazale** [comment]
By the way, I've promoted Kimai to https://ethical.net/resources/

Cheers

**Comment by kevinpapst** [comment]
Never heard of them, but thanks for the sharing the word!!!

**Pull request merged**